euth_wagtail/settings/base.py

```diff
@@ -150,7 +150,8 @@
 
 THUMBNAIL_ALIASES = {
     '': {
-        'organisation_thumbnail': {'size': (80, 80), 'crop': True }
+        'organisation_thumbnail': {'size': (100, 100), 'crop': True },
```

Comment:
May we should rather have `'crop':  'smart'` here. See [easy-thumbnail docs](http://easy-thumbnails.readthedocs.io/en/latest/usage/#thumbnail-options).